### PR TITLE
Publish snapshots from travis

### PR DESCRIPTION
**Pull request by 2m**
## Purpose

Configures the project build for automatic publishing from Travis. For now only snapshots.

## Changes

* added `check` Travis stage that runs various quick checks before running tests
* added sbt-dynver to dynamically determine version
* added sbt-sonatype to publish to sonatype

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,32 +10,62 @@ before_install:
...
+    - script: sbt ++2.13.0-RC2 Test/compile"
+      name: "Compile all tests (with Scala 2.13)"
+    - script: sbt unidoc
+      name: "Create all API docs"
+    - script: sbt docs/paradox
+      name: "Create site with Paradox"
+
     - stage: test
-      script: jabba use $JDK  && java -version && sbt ++$TRAVIS_SCALA_VERSION ";scalafmtCheckAll;test"
+      script: jabba use adopt@~1.8.202-08 && java -version && docker-compose up -d cassandra && sbt test
```
Comment (ennru) [inline .travis.yml]: Shouldn't this run tests for all Scala versions? `+test`?

Comment (2m) [reply .travis.yml]: Good catch.

File: project/Common.scala [hunk 2]
```diff
@@ -1,35 +1,37 @@
...
+    Seq(
+      organization := "com.lightbend.akka",
+      organizationName := "Lightbend Inc.",
+      organizationHomepage := Some(url("https://www.lightbend.com/")),
+      startYear := Some(2016),
+      homepage := Some(url("https://akka.io")),
+      apiURL := Some(url(s"https://doc.akka.io/api/akka-persistence-cassandra/${version.value}")),
+      scmInfo := Some(
+          ScmInfo(
+            url("https://github.com/akka/akka-persistence-cassandra-1.x"),
```
Comment (chbatey) [inline project/Common.scala]: this shouldn't be -1.x should it?

Comment (2m) [reply project/Common.scala]: Yup. Leftovers from work in previous repo. I see that in more places as well. Will fix.

**Comment by ennru** [comment]
Fixes #493

**Comment by ennru** [comment]
Fixes #485

**Pull request merged**